ebuild-maintenance/package-moves/text.xml

```diff
@@ -77,6 +78,9 @@ and update them. These include:
   <li>
     open bug summaries
   </li>
+  <li>
+    wiki articles
```

Comment:
IIUC the technical term is "wiki pages".

Reply:
Thank you!